src/scripts/main.js

```diff
@@ -1,3 +1,294 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('thead');
+
+const rowList = document.querySelectorAll('tbody >tr');
+const tbody = document.querySelector('tbody');
+
+const thead = document.querySelector('thead>tr');
+const headerNames = document.querySelectorAll('thead>tr>th');
+
+for (const nameClicked of headerNames) {
+  nameClicked.isClicked = false;
+};
+
+tableHeader.addEventListener('click', (e) => {
+  const item = e.target.textContent;
+
+  let children = 0;
+
+  for (let i = 0; i < thead.children.length; i++) {
+    if (item === thead.children[i].textContent) {
+      children = i;
+    }
+  }
+
+  const listArr = [...rowList];
+
+  if (e.target.isClicked === false) {
```

Comment:
You can use a logical operator instead
```suggestion
  if (!e.target.isClicked) {
```